.github/workflows/Tests.yaml

```diff
@@ -1089,7 +1091,7 @@ ${{ matrix.build_type }}-pch-${{ matrix.use_pch || 'ON' }}"
           # Clear existing buildcache so we don't accumulate old versions of
           # packages in the cache
           rm -rf $HOME/dependencies
-          spack buildcache create -ufa -m dependencies $SPECTRE_SPACK_DEPS
+          spack buildcache create -uf dependencies $SPECTRE_SPACK_DEPS
```

Comment:
What does this change do?

Reply:
The Spack version is pinned to a specific git hash a few steps up, so nothing should have changed here